packages/core/src/shared/utilities/workspaceUtils.ts

```diff
@@ -360,24 +378,37 @@ export async function collectFiles(
     }
 
     let totalSizeBytes = 0
+
+    const useGitIgnoreFileAsFilter = options?.useGitIgnoreFileAsFilter ?? true
+    const extraExcludeFilePatterns = options?.extraExcludeFilePatterns ?? DefaultExtraExcludePatterns
+    const maxSizeLimitInBytes = options?.maxSizeLimitInBytes ?? maxRepoSizeBytes
```

Comment:
"max" and "limit" are redundant.
```suggestion
    const maxSizeInBytes = options?.maxSizeInBytes ?? maxRepoSizeBytes
```